_pages/en/web-services-access.md

```diff
@@ -21,57 +21,99 @@ This Policy applies only to unclassified assets (data, information, code, ..).
 
 ### Issue
 
-To be able to follow the Policy, ESDC needs better Data Loss Prevention (DLP), including a tool that monitors and enforces classification of files and emails before they can be transmitted on the Internet.
+ESDC is currently blocking access to specific Web 2.0 collaboration tools like Google Docs, Google Hangouts and OneDrive.
+Even though TBS policy generally recommends open access to these tools, ESDC's rationale for blocking them has been primarily to prevent accidental leak of protected information from employees.
+This is a valid concern that must be mitigated with a balanced approach.
 
-Technical Architecture has started to include some SaaS in the ESDC Technology Bricks (GitHub, Trello and Slack), but these don't reflect what can be accessed on the ESDC network or what is currently being used by employees (See [Appendix B](#appendix-b---saas-already-unblocked)).
+The proposed risk mitigation would be to augment the current Data Loss Prevention (DLP) by adding a tool that monitors and enforces classification of files and emails before they can be transmitted on the Internet.
```

Comment:
I'd propably recommend we come up with more diverse options for risk mitigation: DLP for email, OK, but more controlled access to DBs so that people who actually have access to sensitive information/data cannot extract large amounts of data, look for open source options to help monitor key systems use, etc.